packaged/lisk.sh

```diff
@@ -276,6 +276,7 @@ stop_redis_cmd(){
 
 start_lisk() {
 	start_redis
+	setLogspath
```

Comment:
Not sure this warrants a function (not used anywhere else)

Reply:
Sure is not reused but at least it's isolated on it's own